adapters/coap-vertx-quarkus/src/test/resources/application.yml

```diff
@@ -13,7 +13,7 @@
 quarkus:
   log:
     console:
-      color: true
+      color: false
```

Comment:
I'd rather have a consistent setting here in all unit test application.yml files.
Since I usually run the unit tests from the IDE (i.e. not piping mvn test output into a file and viewing it in an editor without color support), I quite like the color output there.

Reply:
Executing a single test in Eclipse IDE will not correctly interpret the ANSI escape characters, thus producing quite a garbled output. However, this can be fixed by installing the [ANSI Escape in Console](https://marketplace.eclipse.org/content/ansi-escape-console) plugin. Also, using colored logs also only works properly when using a dark theme, the default in Eclipse being a light theme with a white background in the console ...
I have no really strong feeling about it and I also do like the colored log on a dark console background :-)
WDYT?

Reply:
@sophokles73 I would be in favor of keeping the color output then. I've added a note about the plugin in the Hono developer documentation (#3253).